backend/app/api/routes/evaluate.py

```
@@ -79,63 +93,70 @@ class ResultResponse(BaseModel):
+                        progress_percent=-1,
+                        message="Evaluation failed",
+                    ),
+                )
+                await handle_evaluation_error(eval_id, str(e))
+            finally:
+                await event_channel.close_channel(eval_id)
+
+        try:
+            task = asyncio.create_task(run_in_background())
+            await register_task(eval_id, task)
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)